plugins/nanospell/plugin.js

```diff
@@ -70,11 +77,11 @@
 				node.getLength() > 0 &&  // and it's not empty
 				( !node.isReadOnly() ) &&   // or read only
 				isNotBookmark(node) && // and isn't a fake bookmarking node
-				// tables and list items can get a bit weird with getNextParagraph()
-				// for example causing list item descendants to be included as part of the original list item
-				// and also individually as their own paragraph-like elements
-				(path.blockLimit ? path.blockLimit.equals(startNode): true) && // check we don't enter another block-like element
-				(path.block ? path.block.equals(startNode): true); // check we don't enter nested blocks (special list case since it's not considered a limit)
+					// tables and list items can get a bit weird with getNextParagraph()
+					// for example causing list item descendants to be included as part of the original list item
+					// and also individually as their own paragraph-like elements
```

Comment:
I think the IDE code formatter thinks this is a continuation of the previous comment.  this is not entirely correct.  will fix or probably move the details of the overly complicated case to below.